### PR TITLE
Add method to init 3d with a view without transition

### PR DESCRIPTION
**Pull request by fgravin**
New method in manger to set up a 3d view without transition.
Parameters are

- lon
- lat
- elevation
- heading
- pitch

### CODE REVIEW

File: src/olcs/contrib/manager.js [hunk 1]
```diff
@@ -245,6 +245,41 @@ olcs.contrib.Manager = class extends ol.Observable {
...
+   *
+   * @export
+   * @param {number} lon
+   * @param {number} lat
+   * @param {number} elevation
+   * @param {number} headingDeg
+   * @param {number} pitchDeg
+   * @returns {Promise<undefined>}
+   */
+  set3dWithView(lon, lat, elevation, headingDeg, pitchDeg) {
```
Comment (gberaudo) [inline src/olcs/contrib/manager.js]: To make it in cleaner, let's pass destination and orientation as parameters here.

Comment (gberaudo) [reply src/olcs/contrib/manager.js]: You are right Cesium is not loaded so disregard my suggestion.

**Comment by fgravin** [comment]
Ok i have tested it further within a project and it works as expected.
The only concern i have is the type of the parameter, should we accept `undefined` or not ? I would say no.

@gberaudo please last check and merge if ok.
Thanks

**Comment by gberaudo** [comment]
Thanks @fgravin, yes the parameter handling is good.

**Pull request merged**